ui/components/fixture-help-wanted-dialog.vue

```diff
@@ -0,0 +1,217 @@
+<template>
+  <app-a11y-dialog
+    id="help-wanted"
+    ref="dialog"
+    :cancellable="state !== `loading`"
+    :shown="context !== null"
+    :title="title"
+    @hide="onHide">
+
+    <template v-if="context !== null">
+
+      <form v-if="state === `ready`" action="#" @submit.prevent="onSubmit">
+        <app-labeled-value
+          v-if="location !== null"
+          :value="location"
+          label="Location" />
+
+        <app-labeled-value
+          v-if="context.helpWanted !== null"
+          :value="context.helpWanted"
+          label="Problem description" />
+
+        <app-labeled-input
+          name="message"
+          label="Message">
+          <app-property-input-textarea
+            v-model="message"
+            :schema-property="{}"
+            name="message" />
+        </app-labeled-input>
+
+        <app-labeled-input
+          name="github-username"
+          label="GitHub username"
+          hint="If you want to be mentioned in the issue.">
+          <app-property-input-text
+            v-model="githubUsername"
+            :schema-property="{}"
+            name="github-username" />
+        </app-labeled-input>
+
+        <div class="button-bar right">
+          <button :disabled="message === ``" type="submit" class="primary">Send information</button>
+        </div>
+      </form>
+
+      <template v-else-if="state === `loading`">
+        Uploading…
+      </template>
+
+      <template v-else-if="state === `success`">
+        Your information was successfully uploaded to GitHub (see the <a :href="issueUrl" target="_blank">issue</a>). The fixture will be updated as soon as your information has been reviewed. Thank you for your contribution!
+
+        <div class="button-bar right">
+          <a href="#" class="button secondary" @click.prevent="$refs.dialog.$emit(`hide`)">Close</a>
```

Comment:
You should be able to emit an `input` event with the string `ready` without a `$ref`. Better extract this into a `close` method anyway.